fastpath/fastpath/core.py

```diff
@@ -1601,6 +1613,39 @@ def flag_measurements_with_wrong_date(msm: dict, msmt_uid: str, scores: dict) ->
         scores["accuracy"] = 0.0
         scores["msg"] = "Measurement start time too old"
 
+def write_measurement_to_disk(msm_tup) -> None:
+    """Write this measurement to disk so that it can be 
+    processed by the measurement uploader
+
+    Args:
+        msm_tup: Measurement tuple as it comes from the work queue
+    """
+    if not conf.write_to_disk:
```

Comment:
I think it's maybe more clear to tell this is a `noop` if we place this check inside of the function that's calling it, that is `msm_processor`.